app/src/main/java/com/android/bookswap/ui/books/add/AddToBook.kt

```diff
@@ -238,53 +234,18 @@ fun AddToBookScreen(
                           .fillMaxWidth(0.5f),
                   enabled = title.isNotBlank() && isbn.isNotBlank(),
                   onClick = {
-                    // Check if title and ISBN are not blank (required fields)
-                    if (title.isNotBlank() &&
-                        isbn.isNotBlank() &&
-                        inputVerification.testIsbn(isbn) &&
-                        selectedGenre != null) {
```

Comment:
Here you removed the verifications, you shouldn't, it is true that two of them are redundant with the enabled field, but the other two should be added.